include/deal.II/particles/particle.h

```diff
@@ -589,7 +590,7 @@ namespace Particles
 
         ArrayView<double> old_properties = this->get_properties();
         ArrayView<double> new_properties =
-          property_pool->get_properties(new_handle);
+          new_property_pool.get_properties(new_handle);
```

Comment:
This was an actual bug (I think introduced just a few weeks ago in #11111), but I think in the old implementation it did not break anything. The particle would use the old property pool to allocated its new properties, but because the properties were just a `double *` both pools could equally access that data. The only problem could be in keeping track of allocated vs freed handles in the two pools. I will modify the particle_handler_18 test to ensure this is fixed (by forcing the old pool to be destroyed before accessing the data in the new pool).

Reply:
Thanks for fixing this!